### PR TITLE
Implement an experimental SVG preview

### LINKED ISSUE #124901: Add SVG support to built-in image preview
Adding the option of viewing a changed SVG as an "rich" image diff instead of the source diff.

### PR DESCRIPTION
**Pull request by ghost**
This PR fixes #124901 by adding SVG support to the `image-preview` extension. Untested and probably unstable.

### CODE REVIEW

File: extensions/image-preview/src/imagePreview.ts [hunk 1]
```diff
@@ -222,7 +245,7 @@ class Preview extends Disposable {
 	<meta name="viewport"
 		content="width=device-width, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0, user-scalable=no">
 
-	<title>Image Preview</title>
+	<title>Image ImagePreview</title>
```
[the remark this answers is not in the document]
Comment (4086606) [reply extensions/image-preview/src/imagePreview.ts]: Resolved in 53f1b12

File: extensions/image-preview/media/main.css [hunk 2]
```diff
@@ -31,39 +31,51 @@ body img {
...
+.container.reflow {
+	border-color: var(--vscode-editorLineNumber-foreground);
+	border-style: solid;
+	border-width: 1px 0 0 1px;
+	margin-top: 9px;
+}
+
+.container.reflow > img {
+	margin: 0 auto auto 0;
+	transform-origin: top left;
```
[the remark this answers is not in the document]
Comment (4086606) [reply extensions/image-preview/media/main.css:78]: Resolved in 53f1b12

**Comment by mjbvz** [comment]
Let's wait for more feedback on the original feature request. There are already some svg preview extension in the marketplace as well

**Comment by 4086606** [comment]
We do have SVG extensions, all of which I regularly use. However, they do not register a custom editor and they do not support a reflow mode for SVGs without a view box

File: extensions/image-preview/package.json [hunk 3]
```diff
@@ -36,41 +43,145 @@
     "customEditors": [
       {
         "viewType": "imagePreview.previewEditor",
-        "displayName": "%customEditors.displayName%",
+        "displayName": "%customEditors.imagePreview%",
         "priority": "builtin",
```
[the remark this answers is not in the document]
Comment (4086606) [reply extensions/image-preview/package.json:48]: Resolved in 23f3aca

**Pull request closed without merging**